### ISSUE TITLE
Форматирование заголовков

### ISSUE THREAD

**Issue by pavelmaliavkin** [pavelmaliavkin/partmart#96]
Вот картинка с примерами заголовков.
![untitled](https://cloud.githubusercontent.com/assets/11453938/11530418/1f885152-98fd-11e5-9925-8e69681cbb23.png)

Пожалуйста, выбери что и где мне нужно использовать (h1 не использовать).

Можешь описать в виде правил, что где использовать.

**Comment by andgam** [comment]
Использовать h3 (только жирный) - везде. А именно:
1. На главной в заголовке к перечню всех запчастей и марок на сайте (+синий цвет)
2. На главной для горячих предложений
3. Во всех попапах для заголовка (контакты, карточка товара, обратная связь)
4. В кабинете и админке - все заголовки
5. Статьи, тарифы и что мы там еще будем добавлять

Само собой это касается только стиля оформления надписи. О тэгах и хедерах для SEO речь отдельно.

**Comment by andgam** [comment]
Аааааа... Ты же видишь, что уродливо вышло. Зачем оставил? Мало ли что я там выбрал, я может неадекват :)

Заголовки делаем ТОЧНО такие как были, один в один. В этой задаче их нужно просто было стандартизировать - например в админке они вразнобой. 

Короче, оставляем старые заголовки, берем за образец любой нормальный заголовок (хоть вот этот) и все заголовки делаем такие же.
![4](https://cloud.githubusercontent.com/assets/11595408/11570402/b17fd758-99ff-11e5-8259-6cabefd58160.png)

**Comment by pavelmaliavkin** [comment]
@andgam 
Андрей, может я не понимаю, что такое стандартизированные?
Сейчас они везде одинаковые.

**Comment by andgam** [comment]
Да, везде одинаковые. Но... ты заставил меня выбрать стиль заголовка, я выбрал h3, а он оказался уродский. Произошло тоже, что с размером шрифта когда мы меняли надпись TechDoc! Я на глаз не умею определять. Потому все время пытаюсь объяснить на пальцах - сделай заголовки такие как были раньше.

Просто раньше они кое-где были нестандартные (например в админке "управление смс") и задача была в том чтобы те НЕСТАНДАРТНЫЕ привести к стандарту, а не выбирать новый стиль заголовка.

Если что - я на телефоне и скайпе. Сорри!

**Comment by pavelmaliavkin** [comment]
Откатили изменения. Будем исправлять локальною
